README.md

```diff
@@ -46,20 +46,48 @@ Follow the instructions in the server's README to get the application server up
     API_KEY = 'SK***'
     API_KEY_SECRET = '***'
 
-
 ### <a name="bullet4"></a>4. Create a TwiML application
 
 Next, we need to create a TwiML application. A TwiML application identifies a public URL for retrieving TwiML call control instructions. When your Android app makes a call to the Twilio cloud, Twilio will make a webhook request to this URL, your application server will respond with generated TwiML, and Twilio will execute the instructions you’ve provided.
 
-To create a TwiML application, go to the TwiML app page. Create a new TwiML application, and use the public URL of your application server’s `/outgoing` endpoint as the Voice Request URL.
+To create a TwiML application, go to the TwiML app page. Create a new TwiML application, and use the public URL of your application server’s `/outgoing` endpoint as the Voice Request URL. 
 
 <img width="700px" src="images/quickstart/create_twml_app.png"/>
 
-
 As you can see we’ve used our ngrok public address in the Request URL field above.
 
 Save your TwiML Application configuration, and grab the TwiML Application SID (a long identifier beginning with the characters "AP").
 
+You can also use `Twilio Functions` to create the TwiML application. Go to the [Functions Page](https://www.twilio.com/console/runtime/functions/manage) and create a new `Functions` by choosing the `Blank` template. Provide `FUNCTION NAME` and `PATH`.
+
+The code snippet below demonstrates a simple TwiML application using `Twilio Functions` that enables making outgoing call to a client or a phone number provided as `server_param_from` argument. 
```

Comment:
The decision to route the call to a client identity or a number is made actually based on the `server_param_to` param. The `server_param_from` is used for the `callerId`.